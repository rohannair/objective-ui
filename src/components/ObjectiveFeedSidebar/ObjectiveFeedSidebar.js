import React, { PropTypes } from 'react'
import styled from 'styled-components'

const ObjectiveFeedSidebar = styled.div`
  flex: 1 1 25%;
`

export default ObjectiveFeedSidebar
