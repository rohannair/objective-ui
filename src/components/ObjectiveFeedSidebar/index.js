import ObjectiveFeedSidebar from './ObjectiveFeedSidebar'
export default ObjectiveFeedSidebar
