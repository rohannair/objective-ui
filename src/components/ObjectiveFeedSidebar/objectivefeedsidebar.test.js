import React from 'react'

import ObjectiveFeedSidebar from './index.js'

describe('Test suite for ObjectiveFeedSidebar component', () => {
  it('ObjectiveFeedSidebar should exist', () => {
    let wrapper = shallow(<ObjectiveFeedSidebar/>)
    expect(wrapper).toMatchSnapshot()
  })
})

