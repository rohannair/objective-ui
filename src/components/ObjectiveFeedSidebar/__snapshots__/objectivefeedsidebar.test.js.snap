exports[`Test suite for ObjectiveFeedSidebar component ObjectiveFeedSidebar should exist 1`] = `
<div
  className="gbVhTM" />
`;
